modules/installation-vsphere-regions-zones.adoc

```diff
@@ -0,0 +1,160 @@
+// Module included in the following assemblies:
+//
+//* installing-vsphere-installer-provisioned-customizations.adoc
+//* installing-vsphere-installer-provisioned-network-customizations.adoc
+//* installing_vsphere/installing-vsphere.adoc
+//* installing-vsphere-network-customizations.adoc
+//* installing-restricted-networks-installer-provisioned-vsphere
+//* installing-restricted-networks-vsphere.adoc
+
+:_content-type: CONCEPT
+[id="installation-vsphere-regions-zones_{context}"]
+= VMware vSphere regions and zones
+
+You can install an {product-title} cluster in a vSphere environment that consists of multiple datacenters running in a single VMware vCenter. 
+
+By specifying multiple datacenters to run in a single VMware vCenter, you can reduce the risk of a hardware failure or network outage causing your cluster to fail.
+
+You can also specify multiple regions and zones on a cluster after installation. 
+
+The following list describes terms associated with defining zones and regions for your cluster:
+
+* Failure domain: Establishes the relationships between a region and zone. You define a failure domain by using vCenter objects, such as a `datastore` object. A failure domain defines the vCenter location for {product-title} cluster nodes.
+* Region: Specifies a vCenter datacenter. You define a region by using a tag from a created tag category, such as `openshift-region`. 
+* Zone: Specifies a vCenter cluster. You define a region by using a tag from from a created tag category, such as `openshift-zone`. 
+
```

Comment:
While you do define region and zone, I think it would be helpful to use an example to illustrate it further.

- Maybe a 3 column table that details example Datacenters, Clusters, Tags. Example of how this can be done using spanned columns (step 4) [1]. Happy to discuss.
- Also - not sure who are official graphics person is, but it would be great to actually get a diagram into the doc, similar to that of slide 5 [2]. I definitely would not gate this work on a diagram, but it would be a nice to have, even if it comes after GA as an enhancement. I suggest posting to `team-ocp-docs` to ask about requesting a diagram.

[1]https://53645--docspreview.netlify.app/openshift-enterprise/latest/installing/install_config/configuring-firewall.html#configuring-firewall_configuring-firewall
[2] https://docs.google.com/presentation/d/1RdRioExW6oAbgfK0DWytR-YHXES2DlHZBpm7x8DTnis/edit#slide=id.g16c2541cbdc_0_0

Reply:
I created a two-tier table. Not sure if a description column would be worthwhile given the definition list/callout list under the default sample config in another section.